web/static/js/reducers/questionnaire.js

```diff
@@ -79,11 +91,14 @@ const deleteChoice = (state, action) => {
   }))
 }
 
-const changeChoice = (state, action) => {
+// TODO: now we're assuming that the choice content being changed is
+// in the default language. Revisit this if we ever allow users to
+// directly edit content in all languages directly from the UI.
+const changeChoice = (state, action, quiz: Questionnaire) => {
```

Comment:
Here: all the references to `defaultLanguage` in the reducer are assuming that we only allow users to edit the current defaultLanguage content. I'm almost sure that this is wrong but refactoring this was way out of reach for the amount of time I had.